Tests/DatadogTests/Datadog/Core/Upload/DataUploadWorkerTests.swift

```diff
@@ -252,7 +252,7 @@ class DataUploadWorkerTests: XCTestCase {
             fileReader: reader,
             dataUploader: mockDataUploader,
             uploadConditions: .alwaysUpload(),
-            delay: DataUploadDelay(performance: UploadPerformanceMock.veryQuick),
+            delay: DataUploadDelay(performance: UploadPerformanceMock.veryQuickInitialUpload),
```

Comment:
This value fits better what we do in this test. We cancel upload worker after the first upload. To eliminate eventual race condition, `.veryQuickInitialUpload` should be used here in the first place (it does 1st upload very quick, and schedules next ones in a distant future).